virt/virtual_machines/vm_networking/virt-defining-an-sriov-network.adoc

```diff
@@ -0,0 +1,20 @@
+[id="virt-defining-an-sriov-network"]
+= Defining an SR-IOV network
+include::modules/virt-document-attributes.adoc[]
+:context: virt-defining-an-sriov-network
+toc::[]
+
+You can create a network attachment for a Single Root I/O Virtualization (SR-IOV) device for virtual machines.
+
+Once the network is defined, you can attach virtual machines to the SR-IOV network.
+
+.Prerequisites
+
+* You must have xref:../../../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[configured an SR-IOV device].
```

Comment:
I'd probably say "You xref:[configured an SR-IOV device for virtual machines]."
It's better to be consistent with the prereq tenses where you can and include as much as the title as possible. If this seems reasonable, please update the rest of the prereq links.

Reply:
Great catch. It's a bit of a rabbit hole with the OCP sriov crossover of modules. Have changed them all to 'You must have..' as that's the most consistent past tense use in the sriov modules, and is more common through the cnv docs.
Have also bumped out the xref text so that it more closely aligns with the title of the linked content.